### PR TITLE
[DO NOT MERGE]enable docker system prune

### PR DESCRIPTION
**Pull request by arroyc**
enabling docker clean up by running docker system prune

### CODE REVIEW

File: vsts/scripts/dockerCleanup.sh [hunk 1]
```diff
@@ -5,7 +5,6 @@
 # --------------------------------------------------------------------------------------------
 
 declare -r BUILD_NUMBER="$BUILD_BUILDNUMBER"
```
Comment (gpcastro) [inline vsts/scripts/dockerCleanup.sh]: what was the motivation for this change?

File: vsts/scripts/dockerCleanup.sh [hunk 2]
```diff
@@ -81,13 +80,9 @@ echo
 docker images
 
 echo
-echo "Cleanup: Run 'docker system prune': $DOCKER_SYSTEM_PRUNE"
```
Comment (gpcastro) [inline vsts/scripts/dockerCleanup.sh]: instead of removing it, we could just set `declare -r DOCKER_SYSTEM_PRUNE="${ORYX_DOCKER_SYSTEM_PRUNE:-true}"`

Comment (arroyc) [reply vsts/scripts/dockerCleanup.sh]: true, but i didn't find any valid use case, do we use this variable during build?

**Comment by gpcastro** [comment]
@arroyc should we close this PR, or is anything pending?

**Pull request closed without merging**